src/vue/navigation/Sidebar.vue

```diff
@@ -166,38 +150,71 @@ export default {
   }
 }
 
-.sidebar__list-item {
+.sidebar__aside {
+  width: $sidebar-width;
+  min-height: 100%;
+  z-index: $z-sidebar;
+  list-style: none;
   display: flex;
-  align-items: center;
-  min-height: 4.8rem;
-  cursor: pointer;
-  color: $col-sidebar-text;
+  flex-direction: column;
 
-  &.router-link-active {
-    background-color: $col-sidebar-active-elem-background;
-    color: $col-sidebar-active-elem-text;
+  @include respond-to-custom($sidebar-hide-bp) {
+    opacity: 1;
+    width: $sidebar-width;
+    background-color: $col-sidebar-background-media-small !important;
```

Comment:
comment on why do we use important

Reply:
they’re redundant mostyl. removed all the other redundant importants